.github/workflows/multinode-inputs.py

```diff
@@ -33,7 +33,8 @@ class Scenario:
 UBUNTU_JAMMY = OSRelease("ubuntu", "jammy", "ubuntu")
 # NOTE(upgrade): Add supported releases here.
 OPENSTACK_RELEASES = [
-    OpenStackRelease("2023.1", "zed", [ROCKY_9, UBUNTU_JAMMY])
+    OpenStackRelease("2024.1", "2023.1", [ROCKY_9, UBUNTU_JAMMY]),
+    OpenStackRelease("2023.1", "zed", [ROCKY_9, UBUNTU_JAMMY]),
```

Comment:
```suggestion
    OpenStackRelease("2023.1", "zed", [ROCKY_9, UBUNTU_JAMMY])
```

Reply:
This is done on purpose 😉 https://peps.python.org/pep-0008/#when-to-use-trailing-commas

Reply:
:exploding_head: